### PR TITLE
Add travis support.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

**Comment by akatsoulas** [comment]
@johngian r?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+language: python
+sudo: false
+python:
+  - "3.3"
+addons:
+  postgresql: "9.3"
+before_script:
+  - psql -c 'create database feedthefox_db;' -U postgres
+install:
+  - pip install -r requirements.txt
```
Comment (johngian) [inline .travis.yml:10]: I would prefer to use peep instead:
Also check:
https://github.com/mozilla/feedthefox/blob/master/tox.ini#L17

It looks like we are installing requirements there.

Comment (johngian) [inline .travis.yml:10]: Also let's remove the `docs` entry from `tox` setup since we don't have any at the moment.

**Comment by johngian** [comment]
Please try this first. In case tests are passing let's merge this.
r+wc

**Comment by akatsoulas** [comment]
Closing this because the site going to be decommissioned soon.

**Pull request closed without merging**